ericsmodule/src/main/java/sdoward/com/ericsmodule/MainActivity.java

```diff
@@ -58,4 +61,6 @@ public void displayBeers(List<Beer> beers) {
         recyclerView.setAdapter(beerAdapter);
 
     }
+
+     public void displaybeerCount (int beerCount) { beerPercentageTextView.setText(String.valueOf(beerCount));}
```

Comment:
you will have to modify this to match the interface when you change it

Reply:
Got it